util/cmake/FindSeqAn.cmake

```diff
@@ -174,10 +174,7 @@ endif ()
 # Require C++11
 # ----------------------------------------------------------------------------
 
-if (NOT (CMAKE_VERSION VERSION_LESS "3.1"))
-    set(CMAKE_CXX_STANDARD 14)
-    set(CMAKE_CXX_STANDARD_REQUIRED on)
-elseif (NOT MSVC) # this implies all compilers within visual studio
+if (NOT MSVC) # this implies all compilers within visual studio
```

Comment:
> And on windows platforms we have no c++14 checks?

MSVC still doesn't define the macro correctly, but supports a large subset of c++14 (we will only using this subset).